torch/ao/quantization/quantizer/xnnpack_quantizer_utils.py

```diff
@@ -487,6 +487,28 @@ def _do_annotate_conv_bn(
     for the following names: "input", "conv", "weight", "bias", and "output".
     """
 
+    # Example inputs for conv-bn1d patterns
+    _conv1d_bn_example_inputs = (
```

Comment:
probably need to remove import of `_conv1d_bn_example_inputs` and `_conv2d_bn_example_inputs` in L12-13 since they are removed from the file